app/components/auth/actions.tsx

```diff
@@ -11,6 +12,7 @@ export function signOut() {
         dispatch({
           type: ACTION_TYPES.AUTH_SUCCEEDED_TO_SIGN_OUT,
         });
+        await dispatch(clearToKeywordSettings());
```

Comment:
I think it's not a good pattern because we don't need to call another action to handle clearing keyword settings.
we can 'listen' the sign out action and 'change state' of keyword settings.

check out the below example.


![image](https://user-images.githubusercontent.com/9796103/66882925-29e7b880-f007-11e9-81d0-470f69b7b6e9.png)